docs/output.md

```diff
@@ -424,6 +426,36 @@ For a Tumor/Normal pair only:
 - `[TUMORSAMPLE].pileup.gz_BAF.txt` and `[NORMALSAMPLE].pileup.gz_BAF.txt`
   - file with beta allele frequencies for each possibly heterozygous SNP position
 
+### MSI status
+
+[Microsatellite instability](https://en.wikipedia.org/wiki/Microsatellite_instability)
+is a genetic condition associated to deficienceies in the
+mismatch repair (MMR) system which causes a tendency to accumulate a high
+number of mutations (SNVs and indels).
+
+#### msisensor
+
+[msisensor](https://github.com/ding-lab/msisensor) is a tool to detect the MSI
+status of a tumor scaning the length of the microsatellite regions. An altered
+distribution of  microsatellite length is associated to a missed replication
+slippage which would be corrected under normal MMR conditions. It requires
```

Comment:
What is MMR standing for?

Reply:
Sorry i must have delete it, it’s mismatch repair